.github/dependabot.yml

```
@@ -1,26 +1,10 @@
 version: 2
 updates:
 
-  # Datatypes crate
   - package-ecosystem: "cargo"
     # Look for `Cargo.toml` and `lock` files in the `root` directory
-    directory: "/datatypes"
+    directory: "/"
     # Check the `crates.io` for updates every day (weekdays)
     schedule:
-      interval: "daily"
-
-  # Operators crate
-  - package-ecosystem: "cargo"
-    # Look for `Cargo.toml` and `lock` files in the `root` directory
-    directory: "/operators"
-    # Check the `crates.io` for updates every day (weekdays)
-    schedule:
-      interval: "daily"
-
-  # Services crate
-  - package-ecosystem: "cargo"
-    # Look for `Cargo.toml` and `lock` files in the `root` directory
-    directory: "/services"
-    # Check the `crates.io` for updates every day (weekdays)
-    schedule:
-      interval: "daily"
+      interval: "live"
```

```suggestion
      interval: "daily"
```

This should correspond to the comment and I think daily is enough.